src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -14,6 +14,16 @@ injections:
         patterns:
           - include: '#wellformed-html'
         end: '(?<=\>)'
+  'L:meta.structure.razor.codeblock - (meta.statement | string.quoted | comment), L:meta.structure.razor.directive.codeblock - (meta.statement | string.quoted | comment)':
```

Comment:
What is `R:` and `L:`? I actually don't understand this entire line.

Reply:
So sorry, I thought I posted my comment here: 

This states, inside of a Razor codeblock or directive block (`@{...}, `@code {...}`, `@functions {...}`) but NOT inside of a string, comment or statement (`@if(..)`) that we inject this grammar. L: indicates, run before any other rule, R: would indicate run after every other rule.